### PR TITLE
Add schema for edge-validator

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This fixes issue #165. I add a jsonschema and a parquet schema for the current edge-validator reporting. The future reports will probably include an id, timestamp, and related environment information.

It seems like testing was broken locally, I'm going to see what CI does in to this PR.

### CODE REVIEW

File: schemas/mozza/edge-validator/edge-validator.1.parquetmr.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+message edge-validator {
+    required group results {
+        required int32 error_count;
+        required float32 error_rate;
+        opetional group errors (MAP) {
```
Comment (acmiyaguchi) [inline schemas/mozza/edge-validator/edge-validator.1.parquetmr.txt]: typo

Comment (mreid-moz) [reply schemas/mozza/edge-validator/edge-validator.1.parquetmr.txt]: I think this typo is what's causing the tests to fail.

**Comment by mreid-moz** [comment]
I talked with some of the other folks on the team about putting "internal use" schemas in the `mozza` namespace, and generally people thought it wasn't a great idea. 

Let's make a separate namespace for `edge-validator`, then name these pings as `error-report` or something similarly descriptive.

**Comment by acmiyaguchi** [comment]
I changed the namespace and fixed the parquet schema to reflect the jsonschema. The tests should all pass now.

File: validation/edge-validator/error-report.1.sample.fail.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "results": {
+        "namespace.doctype.1": {
+            "error_count": -1,
```
Comment (mreid-moz) [inline validation/edge-validator/error-report.1.sample.fail.json]: Let's split the various failure cases into separate example pings (negative error count, >100 error rate, missing errors) to make sure any of the three cases cause the expected failure.

**Pull request merged**